jerry-core/parser/js/js-parser.c

```diff
@@ -1489,6 +1489,10 @@ parser_post_processing (parser_context_t *context_p) /**< context */
 
   compiled_code_p = (ecma_compiled_code_t *) parser_malloc (context_p, total_size);
 
+#ifdef JMEM_STATS
+  jmem_stats_allocate_byte_code_bytes (total_size);
+#endif /* JMEM_STATS */
```

Comment:
parser_malloc is used at other places, too, but they don't seem to be stat'd. moreover, parser_free is not stat'd anywhere. is that intentional?